get_url/help.md

```diff
@@ -1,18 +1,18 @@
 # Description
 
-The Get URL InsighConnect plugin allows you to download files from a URL. Supported protocols are HTTP, HTTPS, and FTP.
+The Get URL InsightConnect plugin allows you to download files from a URL. Supported protocols are HTTP, HTTPS, and FTP.
```

Comment:
We don't say InsightConnect in other places and it's a label on the Hub page.

```suggestion
The Get URL plugin allows you to download files from a URL. Supported protocols are HTTP, HTTPS, and FTP.
```

Reply:
fixed